app/core/models.py

```diff
@@ -1,3 +1,24 @@
 from django.db import models
+from django.contrib.auth.models import User as DjangoUser
 
-# Create your models here.
+
+class User(DjangoUser):
```

Comment:
for each model add Meta class with str representation and for Event model add ordering set to organize Event's objects with date